veros/setup/acc2/acc2.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
This setup is identical with `ACC` and thus redundant. Please remove it from the PR.